edgelet/iotedge/src/config/import/old_config/mod.rs

```diff
@@ -209,7 +209,4 @@ homedir: '/var/lib/iotedge'
 moby_runtime:
   uri: 'unix:///var/run/docker.sock'
   network: 'azure-iot-edge'
-
```

Comment:
This was the old default, so keep it.

Reply:
Fixed. I also reflected the change in the import test files